### PR TITLE
Removed the schellingCoin deployment code

### PR DESCRIPTION
**Pull request by adi44**
Currently, I have commented on the two lines which execute the deployment of SchellingCoin.sol. So if in the future we want to deploy again we can remove the comment.

### CODE REVIEW

**Comment by SkandaBhat** [comment]
@adi44 Remove instead of commenting, we can always bring it back using git history.

**Comment by dev1644** [comment]
> @adi44 Remove instead of commenting, we can always bring it back using git history.

Agreed with @SkandaBhat here, that's why the `Schelling coin` deployment was added to the last so it can be removed with out changing the migration process much.

@adi44 You can delete `Schelling coin` deployment & `9_delpoy_schelling_coin.js` file.

File: migrations/deploy_all.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ async function main() {
   await voteManagerMigration();
   await delegatorMigration();
   await faucetMigration();
-  await schellingCoinMigration();
+  //await schellingCoinMigration();
```
Comment (dev1644) [inline migrations/deploy_all.js:23]: @adi44 You can delete `Schelling coin` deployment here & also `9_delpoy_schelling_coin.js` file in `/migrations/src`.

**Review by SkandaBhat — DISMISSED** [review]
LGTM

File: migrations/deploy_all.js (hunk 1 above)
Comment (adi44) [reply migrations/deploy_all.js]: @dev1644 I have done the needful

**Comment by dev1644** [comment]
Hi @adi44 , currently changes are not too big and also have same objective.
It would be better if you can roll (squash) them into one commit instead of three.

**Comment by adi44** [comment]
@dev1644 I have done the needful

**Review by dev1644 — APPROVED** [review]

**Review by dev1644 — APPROVED** [review]

**Pull request closed without merging**